DTXManiaプロジェクト/コード/全体/CConfigIni.cs

```diff
@@ -2936,5 +2998,25 @@ private void tデフォルトのキーアサインに設定する()
 		}
 		//-----------------
 		#endregion
+
+
+	    public event PropertyChangedEventHandler PropertyChanged;
+
+	    private bool SetProperty<T>(ref T storage, T value, string propertyName = null)
```

Comment:
Here is that reusable property setting and event raising code I mentioned elsewhere.